packages/js/__tests__/jsonrpc.ava.js

```diff
@@ -2,7 +2,7 @@ import process from 'process';
 
 import test from 'ava';
 
-import {JsonRpcProvider} from '..';
+import {JsonRpcProvider} from '../dist/index.js';
```

Comment:
@ailisp we needed to make this small change in order to make it compatible with the lint